src/components/20-molecules/datepicker/e2e.js

```diff
@@ -36,5 +36,55 @@ test.describe('datepicker', () => {
         '29.02.2020 (of 1 submittable elements)'
       );
     });
+
+    test('should just add years of a given range', async ({ page }) => {
+      await page.goto(fixtureURL('components-datepicker--datepicker'));
+
+      const datepicker = await page.locator('axa-datepicker');
+      await datepicker.evaluate(node => {
+        node.allowedyears = ['1999-2000'];
+      });
+
+      const datepickerYearDropdown = await page.locator(
+        'axa-datepicker .js-datepicker__dropdown-year'
+      );
+      const itemsString = await datepickerYearDropdown.getAttribute('items');
+
+      const itemsArray = JSON.parse(itemsString);
+
+      expect(itemsArray.length).toBe(2);
+      expect(itemsString).toContain('"value":"1999"');
```

Comment:
This was straightforwardly copied from the old Testcafe test. But now that we have an opportunity to redo it, we can do better: use `itemsArray[0]` and `itemsArray[1]` and test (toBe) for the exact values at the exact positions!